frontend/src/pages/Overview/OverviewPage.tsx

```diff
@@ -744,7 +744,7 @@ export class OverviewPage extends React.Component<OverviewProps, State> {
                   const isLongNs = ns.name.length > NS_LONG;
                   return (
                     <GridItem sm={sm} md={md} key={'CardItem_' + ns.name} style={{ margin: '0px 5px 0 5px' }}>
-                      <Card isCompact={true} className={cardGridStyle}>
+                      <Card isCompact={true} className={cardGridStyle} data-namespace={ns.name}>
```

Comment:
Note that the code will need to add more of these IDs to help make cleaner UI tests.
In the past QE asked Dev to add these ids, but any QE eng is free to add these IDs if that helps to simplify the selectors or not to iterate on the DOM to find a specific element.

In most of the cases those are transparent to the logic.

Reply:
+1 more of these should be added in the future to aid testing and they can be added by whoever is writing the test

Reply:
Yes, I'd say that when let's use it when writing the test, just to avoid that some selector/expression is complex due to a lack of data-* (i.e. "find a div with a class, search the first element of a list, iterate until the proper sibling, etc", if a "data-*" would simplify that, it's something we can use without worries).

Reply:
Yes, the "data-*" attributes depend on the domain (Kiali) and I expect that the author of the test can add them to simplify the testing.

In the past, typically QE asked Dev to add some Ids for better selection handling, but I don't see that necessary (unless there is some complex corner case).